Frameworks/CoreGraphics/CGColor.mm

```diff
@@ -27,21 +27,17 @@
 static IWLazyClassLookup __LazyUIColor("UIColor");
 
 /**
- @Status Caveat
- @Notes Limited constants supported
+ @Status Interoperable
 */
 CGColorRef CGColorGetConstantColor(CFStringRef name) {
     CGColorRef ret = nullptr;
-    woc::StrongCF<CGColorSpaceRef> colorspace;
+    woc::StrongCF<CGColorSpaceRef> colorspace{ woc::MakeStrongCF<CGColorSpaceRef>(CGColorSpaceCreateDeviceGray()) };
     if (CFEqual(kCGColorBlack, name)) {
         ret = static_cast<CGColorRef>([[__LazyUIColor blackColor] CGColor]);
-        colorspace = woc::MakeStrongCF<CGColorSpaceRef>(CGColorSpaceCreateDeviceGray());
     } else if (CFEqual(kCGColorWhite, name)) {
         ret = static_cast<CGColorRef>([[__LazyUIColor whiteColor] CGColor]);
-        colorspace = woc::MakeStrongCF<CGColorSpaceRef>(CGColorSpaceCreateDeviceGray());
     } else if (CFEqual(kCGColorClear, name)) {
         ret = static_cast<CGColorRef>([[__LazyUIColor clearColor] CGColor]);
-        colorspace = woc::MakeStrongCF<CGColorSpaceRef>(CGColorSpaceCreateDeviceGray());
     } else {
         UNIMPLEMENTED_WITH_MSG("CGColorGetConstantColor does not support color %s", CFStringGetCStringPtr(name, kCFStringEncodingUTF8));
```

Comment:
Nit: Unknown constant color name should be what we trace here, also not through UNIMPLEMENTED macros, since we support all documented names.